src/context/AppProvider.tsx

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React from "react";
 
-import ThemeProvider from './ThemeProvider';
+import ThemeProvider from "./ThemeProvider";
+import TreeContextProvider from "./TreeContext";
 
 export default function AppProvider({ children }: AppProps) {
-	return (
-		<React.Fragment>
-			<ThemeProvider>{children}</ThemeProvider>
-		</React.Fragment>
-	);
+  return (
```

Comment:
We don't get anything inside the `{}`, so I wonder if we can refactor our code as 


```ts
const AppProvider: React.FC = ({children}) => (
  // code
);
```

Reply:
Oh great, I never thought of that.